packages/backend-core/src/db/utils.ts

```diff
@@ -416,11 +422,11 @@ export const getPlatformUrl = async (opts = { tenantAware: true }) => {
       platformUrl = platformUrl.replace("://", `://${tenantId}.`)
     }
   } else if (env.SELF_HOSTED) {
-    const db = getGlobalDB()
+    const dbx = db ? db : getGlobalDB()
```

Comment:
I think the `db` here will always be global db, and it's no extra work to retrieve the db instance using the `getGlobalDB` function as that's initialised as a part of the tenancy middleware.

To keep the method signature of `getPlatformUrl` more clean we can remove the passed in `db` arg.

Reply:
Fair enough, I've set this to a fixed call to the global db and cleared out any references to the old signature.